wsgi/odyssy/news/models.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.shortcuts import get_object_or_404
+
+# Create your models here.
+
+
+class News(models.Model):
+    """ Model of News app
```

Comment:
Done, made changes according to https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings